app/views/audit.scala.html

```diff
@@ -17,6 +20,71 @@
     <link rel="stylesheet" href='@routes.Assets.at("javascripts/SVLabel/build/SVLabel.css")'/>
     <link rel="stylesheet" href='@routes.Assets.at("stylesheets/animate.css")'/>
     <div id="page-loading"><img id="loading-gif" src='@routes.Assets.at("assets/page-load.gif")'/></div>
+    @if(user){
+        @if(user.get.roles.getOrElse(Seq("")).contains("User")){
+            <div class="modal fade" id="survey-modal-container" tabindex="-1" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true">
+                <div class="modal-dialog">
+                    <div class="modal-content" id="survey-modal">
+                        <div class="modal-header">
+                            <h4 class="modal-title" id="sign-in-label">Feedback Survey</h4>
+                        </div>
+                        <div class="modal-body">
+
+                            <form id="surveyForm" method="post">
+                                <ol>
+                                @for(surveyQuestion <- SurveyQuestionTable.listAll){
```

Comment:
So each time we need to modify the survey, we need to modify the database records, correct?

Reply:
We can just set delete=true for the questions we dont want and/or add new questions to the table. Ideally we'll also have a role field for the questions to indicate who should see which questions and in what order.